src/objects/weather/fog.js

```diff
@@ -0,0 +1,45 @@
+class Fog {
+  constructor({ game, config }) {
+    this.game = game;
+    this.config = config;
+
+    this.defaults = {
+      alpha: 0.7,
+      color: '#b2ddc8',
```

Comment:
Should we abstract all colors to the config?

Reply:
Good point, will change it. We could probably move all the defaults to the config under some weather.defaults prop or the like.